File: Semantic Annotator/FormsForJSON-LD/src/main/java/bridgepattern/BridgePattern.java
package main.java.bridgepattern;

import java.util.ArrayList;

public class BridgePattern {
	
	public ArrayList<String> triples;
	public ArrayList<String> query;
	public ArrayList<String> entities;
	public ArrayList<String> sen_var_ent;
	
	private String part;
	
	
	
	public BridgePattern() {
		triples = new ArrayList<String>();
		query = new ArrayList<String>();
		entities = new ArrayList<String>();
		sen_var_ent = new ArrayList<String>();
	}



	public String getPart() {
		return part;
	}



	public void setPart(String part) {
		this.part = part;
	}
	
	

}

File: Semantic Annotator/src/main/python/BridgePattern.py
class BridgePattern(object):
    id = ""
    part = ""
    query = []
    
    entities = []
    sen_var_ent = []
    triples = []

    # The class "constructor" - It's actually an initializer 
    def __init__(self, id, part, query, entities, sen_var_ent, triples):
        self.id = id
        self.part = part
        self.query = query
        
        self.entities = entities
        self.sen_var_ent = sen_var_ent
        self.triples = triples

def makeBridgePattern(id, part, query, entities, sen_var_ent, triples):
    bridgePattern = BridgePattern(id, part, query, entities, sen_var_ent, triples)
    return bridgePattern